### PR TITLE
Feature/assets folder

### PR DESCRIPTION
**Pull request by SanyamSamS**

### CODE REVIEW

**Review by JohnM89 — COMMENTED** [review]
looks good

**Comment by JohnM89** [comment]
looks good

**Review by justinsta624 — COMMENTED** [review]
Good job!

**Review by JohnM89 — APPROVED** [review]

File: assets/script.js
Comment (justinsta624) [inline assets/script.js:1]: looks good to me

File: assets/style.css
Comment (justinsta624) [inline assets/style.css:1]: looks fine

**Pull request merged**